app/Models/Loan.php

```diff
@@ -30,6 +30,7 @@ class Loan extends Model
      * @var array
      */
     protected $fillable = [
+        'id',
```

Comment:
PK rather shouldn't be fillable

Reply:
Well, I agree with your opinion, it just happened on this model, I was too quick to copy it all!